src/ClientAssetsManager.php

```diff
@@ -186,6 +187,9 @@ private function initJavaScriptOptimization()
                 /** @var _WP_Dependency $dependency */
                 $dependency = $wpScripts->registered[$script];
                 $path = $dependency->src;
+                if ($path === false) {
```

Comment:
This change is actually #4 (can be seen [here](https://github.com/FlyingDR/wordpress-client-assets-manager/pull/4/files#diff-f7a121a14fef668a0969fe2a391526e9cf8554c1d1a6b8812e7d6b703977129bR189)), so it should not appear as part of another pull request